### ISSUE TITLE
Issue on page /02-Randomised-Experiments.html

### ISSUE THREAD

**Issue by y1-y0** [matheusfacure/python-causality-handbook#84]
A bit confused by wording. Seems like students were, after all, randomized.

> They’ve randomized not the students, but the classes. Some of them were randomly assigned to have face-to-face lectures, others, to have only online lectures and a third group, to have a blended format of both online and face-to-face lectures.

**Comment by matheusfacure** [comment]
Thks for the feedback. I tried to rephrase it to make it more clear. Should be fixed once I release the new chapter
